.gitignore

```diff
@@ -55,10 +55,3 @@
 *.fatbin
 *.cu.deps
 *.cu.cache
-
-
-#build 
-/build/*
-
-#editors/IDEs
-/.vscode/*
```

Comment:
I know these are unrelated to issue #538, but i think it'd be good if .gitignore included them. It's good not having to worry about staging build for instance. Are you interested in another PR that adds some common stuff to the .gitignore?

Reply:
@gkarlos please create a separate PR for this change if you think that this is valid :)